mantle/cmd/kola/kola.go

```diff
@@ -648,7 +649,17 @@ func getParentFcosBuildBase(stream string) (string, error) {
 			return getParentFcosBuildBase("testing-devel")
 		}
 
-		parentVersion = index.Releases[n-1].Version
+		findParentVersion:
```

Comment:
let's add a comment above this block.. i.e. something along the lines of:

Now that we build for multiple architectures we are allowing failures for secondary architectures to be non-fatal and promoting them anyway on non-prod streams. This means that an architecture can be missing from that particular release. Let's search the releases in reverse order to find the most recent release that does have this architecture.

Reply:
addressed in recent commit. could you please have a look?